Dockerfile.dev

```diff
@@ -0,0 +1,17 @@
+FROM node:16-alpine
+
+WORKDIR /app
+
+COPY package.json ./
+
+COPY pnpm-lock.yaml ./
+
+RUN npm install pnpm -g
+
+RUN pnpm install
+
+COPY . .
```

Comment:
can we remove this copy command. if you are using this file for development you need to rebuild the image every time when you make a code change. Instead of copying you can mount the source code to the container via docker-compose. It will retain the dev mode without needing for a rebuild.

Reply:
Gave it a thought, i think having copy is ok. Later we can mount the /src. WDYT @akhilmhdh

Reply:
Copy is ok. It kinda keeps close to original process. But yah app container needs to be mounted to detect file changes